app/controllers/users/sessions_controller.rb

```diff
@@ -50,5 +63,11 @@ def track_authentication_attempt(email)
 
       analytics.track_event('Authentication Attempt with nonexistent user')
     end
+
+    def password
+      params.require(:user)[:password]
+    rescue ActionController::ParameterMissing
+      ''
```

Comment:
I'm not convinced we need this test. I don't think we have any controller tests that don't pass in the required params because it is assumed that the form is properly configured. To test our form, we would need a feature spec, and if our form is broken, the test would fail, which would prompt us to fix the form.